BlueprintUICommonControls/Sources/AttributedLabel.swift

```diff
@@ -96,8 +96,25 @@ public struct AttributedLabel: Element, Hashable {
         .foregroundColor: UIColor.systemBlue.withAlphaComponent(0.3),
     ]
 
+    /// This is the fallback value used when the environment has no URL handler
+    private var defaultURLHandler: URLHandler
+
+    @available(iOSApplicationExtension, unavailable, message: "This cannot be used in application extensions")
     public init(attributedText: NSAttributedString, configure: (inout Self) -> Void = { _ in }) {
```

Comment:
I assume we can't then do something like: 

```
    @available(iOSApplicationExtension, available)
    public init(attributedText: NSAttributedString, configure: (inout Self) -> Void = { _ in }) {
        self.init(
            attributedText: attributedText,
            defaultURLHandler: AnExtensionSafeURLHandler(),
            configure: configure
        )
    }
```

And avoid the new init?

Reply:
I _think_ you'd need to inject a `NSExtensionContext` somehow in order for that to work. Is there a way to get the `NSExtensionContext` without piping it through?

Reply:
Best I can tell, the only way to get an `NSExtensionContext` is via a UIViewController.
Further, that API is heavily restricted such that you can only open your host app.